src/components/Wallets/Wallets.js

```diff
@@ -32,12 +32,13 @@ const {
 class Wallets extends PureComponent {
   constructor(props) {
     super(props)
+    const { currentTime } = props
     this.handleDateChange = this.handleDateChange.bind(this)
```

Comment:
one file missing here 😆

Reply:
@jarindr i was just slightly changing that component, of course i can refactor every bind in the project, there are a lot of them, but is it needed? :)

Reply:
not really :P